src/Dialog.js

```diff
@@ -94,15 +95,17 @@ const Dialog = ({children, ...props}) => {
 Dialog.defaultProps = {theme}
 
 Dialog.propTypes = {
+  ...COMMON.propTypes,
   ...LAYOUT.propTypes,
-  ...systemPropTypes.space,
-  ...systemPropTypes.color,
   children: PropTypes.node.isRequired,
   isOpen: PropTypes.bool.isRequired,
   onDismiss: PropTypes.func.isRequired,
+  ...sx.propTypes,
   theme: PropTypes.object
 }
 
+Dialog.displayName = 'Dialog'
```

Comment:
Do you remember why this is needed?

Reply:
`Dialog` had no display name, possibly due to the way it is being assigned. I didn't look into it too carefully.